accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -515,6 +515,24 @@ void main() {
 
 The grammar changes necessary for these changes are provided separately (as [changes to the spec grammar](https://dart-review.googlesource.com/c/sdk/+/197161)). The grammar change examples above are for illustration only.
 
+### Constant expressions
+
+We add the following to the set of expressions that are potentially constant and constant:
+
+If `e` is a potentially constant expression, `T1..Tk` is derived from `<typeList>`, and `e<T1..Tk>` can be derived from `<primary> <selector>*`, then `e<T1..Tk>` is a potentially constant expression.
+If moreover `e` is a constant expression whose static type is a function type `F`, and `T1..Tk` is a list of constant type expressions, then `(e)<T1..Tk>` is a constant expression.
```

Comment:
Drop the parentheses in `(e)`, it's constant without the parentheses too (and since parentheses preserve both constness and type, it's also constant with parentheses).

Reply:
Of course! Done. (I removed those `()` in a few other locations, but forgot this one.)